nixos/modules/services/network-filesystems/kbfs.nix

```diff
@@ -0,0 +1,80 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.kbfs;
+
+in {
+
+  ###### interface
+
+  options = {
+
+    services.kbfs = {
+
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to mount the Keybase filesystem.";
+      };
+
+      mountPoint = mkOption {
+        type = types.str;
+        default = "/keybase";
```

Comment:
As this service is expected to be used by multiple users, defaulting to ```/keybase``` may not be the best thing. I would recommend ```"%h/keybase"``` instead.